backend-deno/samples/logger/mod.ts

```diff
@@ -0,0 +1,27 @@
+import { startService } from "https://deno.land/x/telestion/mod.ts";
+import { encode } from "https://deno.land/std@0.186.0/encoding/hex.ts";
+
+const encoder = new TextEncoder();
+
+const { messageBus } = await startService();
+
+const logMessages = messageBus.subscribe("log.>");
+
+console.log("Logger started");
+
+for await (const msg of logMessages) {
+  try {
+    const currentTime = new Date().toISOString();
+    const logMessage = encode(msg.data).toString();
+    const subject = msg.subject.split(".")[1];
+
+    console.log(`${currentTime} [${subject}] ${logMessage}`);
+    await Deno.writeFile(
+      "log.txt",
```

Comment:
WHile this should use the `DATA_DIR` later, this is tracked separately in TELESTION-473 as the current behavior matches that of the example in the documentation, which this sample should be equivalent to.